suite-common/graph/src/graphBalanceEvents.ts

```diff
@@ -107,19 +112,27 @@ export const getAccountMovementEvents = async ({
     account,
     startOfTimeFrameDate,
     endOfTimeFrameDate,
+    dispatch,
 }: {
     account: AccountItem;
     startOfTimeFrameDate: Date | null;
     endOfTimeFrameDate: Date;
+    dispatch: ReturnType<typeof useDispatch>;
 }) => {
     const { coin, identity, descriptor } = account;
 
     const getBalanceHistory = async () => {
         if (getNetworkType(coin) === 'ripple') {
-            return TransactionCacheEngine.getAccountBalanceHistory({
+            const allTransactions = await dispatch(
+                fetchAllTransactionsForAccountThunk({
+                    accountKey: account.accountKey,
+                }),
+            ).unwrap();
+
+            return getAccountHistoryMovementFromTransactions({
+                transactions: allTransactions,
                 coin,
-                descriptor,
-            });
+            }).main;
```

Comment:
Please add comment why only `main` is needed and tokens are not.

Reply:
done, it's because we don't have UI for multiple movements in one graph, will be fixed in next PR